src/models/lineChart.js

```diff
@@ -383,7 +383,8 @@ nv.models.lineChart = function() {
                 var singlePoint, pointIndex, pointXLocation, allData = [];
                 data
                     .filter(function(series, i) {
-                        series.seriesIndex = i;
+                        // Assign the index here instead of in the filter
+                        // as the DOM removes disabled series
```

Comment:
this comment says your'e assigning the index here, but you moved it away from here...  ?